metrics.nim

```diff
@@ -161,7 +161,7 @@ when defined(metrics):
       if label in invalidLabelNames:
         raise newException(ValueError, "Invalid label: '" & label & "'. It should not be one of: " & $invalidLabelNames & ".")
 
-  proc ignoreSignalsInThread() =
+  proc metricsIgnoreSignalsInThread*() =
```

Comment:
This rename was done to avoid conflict with `ignoreSignalsInThread()` defined in `nat.nim`.